ops/cli.py

```diff
@@ -1,351 +1,343 @@
+        else:
+            error(f"GKE cluster {self.cluster_name} not found")
+            info(f"    Available clusters in {self.project}:", plain=True)
+            info(shell(f"gcloud --project {self.project} container clusters list", quiet=True), plain=True)
+            issues += 1 if self.force else exit(1)
+        if self.__namespace_exists():
+            success(f"Namespace {self.cluster_namespace} exists in cluster")
+            shell(f"gcloud set-context --current --namespace={self.cluster_namespace}")
+            info("    Namespace now set")
+        else:
+            error(f"Namespace {self.cluster_namespace} not found in cluster")
+            info(f"    Available namespaces in {self.cluster_name}:", plain=True)
+            info(shell("kubectl get namespace", quiet=True), plain=True)
+            issues += 1 if self.force else exit(1)
+        if issues != 0:
+            warn(f"{issues} issues but continuing due to '--force'")
+        else:
+            success("Validated configuration")
+
+    def __cluster_exists(self) -> bool:
+        """Check that the cluster exists."""
+        clusters = json.loads(shell(f"gcloud --project {self.project} --format=json "
+                                    f"container clusters list "
+                                    f'{f"--zone {self.cluster_zone}" if self.cluster_zone else ""}'))
+        return self.cluster_name in [c["name"] for c in clusters]
+
+    def __cloudsql_exists(self) -> bool:
+        """Check that the Cloud SQL instanc exists."""
+        instances = json.loads(shell(f"gcloud --project {self.project} --format=json "
+                                     f"sql instances list"))
+        return self.cloudsql_name in [i["name"] for i in instances]
+
+    def __namespace_exists(self) -> bool:
+        """Configure K8s context and check that the namespace is present."""
+        info(f"=>  Configuring Kubernetes for cluster {self.cluster_name}")
+        shell(f"gcloud --project {self.project} container clusters get-credentials "
+              f"{self.cluster_name} --zone {self.cluster_zone}")
+        ctx = f"gke_{self.project}_{self.cluster_zone}_{self.cluster_name}"
+        shell(f"kubectl config use-context {ctx}")
+        try:
+            namespaces = json.loads(shell("kubectl get namespace -o json", timeout=5))
+            return self.cluster_namespace in [item["metadata"]["name"] for item in namespaces["items"]]
+        except subprocess.TimeoutExpired:
+            error("Namespace query took too long--maybe you need to be on non-split VPN?")
+            return False
+
+
+class InfoCommand(Config):
+    """Class for info command that doesn't do anything special, just resolves/validates arguments."""
+
+    def __init__(self, parsed_args: argparse.Namespace):
+        super().__init__(parsed_args)
+
+    # This can't be static if we're to override it in subclasses
+    # noinspection PyMethodMayBeStatic
+    def execute(self) -> int:
```

Comment:
This is broken - you've had to work around your design already.

Reply:
If Python was properly typed I'd need this, but since it isn't who cares. I swapped out the lint ignore with `@staticmethod` since that's more clear, it doesn't cause any issues because Python is none the wiser

Reply:
> since it isn't who cares

Why have you made this change then??

Reply:
Resolved in new version. No methods exist in the file and `typing` has been important to type the core components (config and command_mapping).